docs/program-setup/mbed_targets.md

```diff
@@ -194,7 +194,9 @@ Enabling `is_disk_virtual` adds delay after flashing firmware binary to make sur
 
 ### supported_toolchains
 
-The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `uARM`, `ARMC5`, `ARMC6`, `GCC_ARM` and `IAR`.
+The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `ARMC5`, `UARM`, `ARMC6`, `GCC_ARM` and `IAR`.
+
+<span class="notes">**Note**: Mbed OS 6 does not support Arm Compiler 5 or uARM.</span>
 
 When using `ARM`, `ARMC5`, `ARMC6` for `supported_toolchains`, please note:
```

Comment:
Should we not get rid of every mentions of `ARMC5` expect to say it is no longer supported?

Reply:
I did that at first.  But if you look at the list of values, is it still there? That may confuse people.

Reply:
The list of values that can go in supported_toolchains. Is there anywhere you can query that list to see all the values, and will that query return uArm and ARMC5?